game.py

```diff
@@ -20,7 +21,36 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    
+    
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+
+    wrong_guesses_list = []
+    
+    while (len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES 
+           and not is_word_guessed(snowman_word, correct_letter_guess_statuses)): 
+    
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        
+        if user_input in correct_letter_guess_statuses:
```

Comment:
Note that the `if user_input in correct_letter_guess_statuses` is shorthand ([in and not in](https://docs.python.org/3/reference/expressions.html#in)) for: 
```py
for element in sequence: 
    if element == sought_after: 
        return False
    return True
```

You can use this syntax to perform the above logic over a multiple kinds of iterables/sequences in Python. Later in the curriculum we will learn that they don't all have the same performance (Big O).